gcs-app/gcs.py

```diff
@@ -0,0 +1,43 @@
+import logging, os, random, string, sys
+
+from google.cloud import storage
+
+class Gcs(object):
+    """
+    This class handles the logic for reading and writing to a GCS bucket.
+    """
+    def __init__(self):
+        # Set logging
+        logging.basicConfig(stream=sys.stdout, level=logging.INFO)
+        # Setup GCS configs
+        self.storage_client = storage.Client()
+
+    def write_to_file(self, file_name, string_length):
+        letters = string.ascii_letters
+        message_to_file = ''.join(random.choice(letters) for i in range(string_length))
+        print("Random string is:", message_to_file)
+
+        text_file = open(file_name, "w")
+        n = text_file.write(message_to_file)
+        text_file.close()
+        return text_file
+
+    def upload_blob(self, bucket_name, file_name, string_length):
+        """Uploads a file to the bucket."""
+        self.write_to_file(file_name, string_length)
```

Comment:
I don't see why a GCS class would have a function called `write_to_file` that has nothing to do with GCS.  If you're going to make a class, you need to make a class with well-intentioned functions.

In this case, `upload_blob` should take a blob or a string as contents, not `string_length` (see above comment). It also doesn't need to call another function to write to a file, since that function would never be used by itself.  You should simply call `blob.upload_from_string()` and forget about files altogether.